Task 2/Task 2.1.2/Task 2.1.2/Program.cs

```diff
@@ -0,0 +1,804 @@
+                    Console.WriteLine("Введите координаты вершины прямоугольника и длины его сторон.");
+                    Rectangle rectangle = new Rectangle(InputCoordinats(), InputCoordinats(), CheckingValues(0), CheckingValues(0));
+                    Console.WriteLine("Прямоугольник создан.");
+                    figures.Add(rectangle);
+                    CustomPaint(name, figures);
+                    break;
+                case Figures.Треугольник:
+                    Console.WriteLine("Введите координаты вершины треугольника и длины его сторон.");
+                    Triangle triangle = new Triangle(InputCoordinats(), InputCoordinats(), CheckingValues(0), CheckingValues(0), CheckingValues(0));
+                    Console.WriteLine("Проверка треугольника на валидность.");
+                    IsTriangleValid(triangle);
+                    Console.WriteLine("Треугольник создан.");
+                    figures.Add(triangle);
+                    CustomPaint(name, figures);
+                    break;
+                case Figures.Сфера:
+                    Console.WriteLine("Введите координаты центра сферы и её радиус:");
+                    Sphere sphere = new Sphere(InputCoordinats(), InputCoordinats(), CheckingValues(0));
+                    Console.WriteLine("Сфера создана.");
+                    figures.Add(sphere);
+                    CustomPaint(name, figures);
+                    break;
+                case Figures.Цилиндр:
+                    Console.WriteLine("Введите радиус и высоту цилиндра");
+                    Cylinder cylinder = new Cylinder(0, 0, CheckingValues(0), CheckingValues(0));
+                    Console.WriteLine("Цилиндр создан.");
+                    figures.Add(cylinder);
+                    CustomPaint(name, figures);
+                    break;
+                default:
+                    Console.WriteLine("Такой команды не существует! Введите другую команду.");
+                    CustomPaint(name, figures);
+                    break;
+            }
+        }
+
+        /// <summary>
+        /// Метод, использующися для ввода координат точки фигуры. Например, центр курга, конечной точки линии или вершины треугольника.
+        /// </summary>
+        /// <returns>Возвращает float значение координаты</returns>
+        static float InputCoordinats ()
+        {
+            Console.WriteLine("Введите координату (только число):");
+            do
+            {
+                string usercoordinate = Console.ReadLine();
+                if (float.TryParse(usercoordinate, out float coordinate))
+                    return coordinate;
+            }
+            while (true);
+        }
+
+        /// <summary>
+        /// Метод, использующися для проверки введеных значений.
+        /// </summary>
+        /// <returns>Возвращает удовлетворяющее условиям проверки float значение</returns>
+        static float CheckingValues (float checkvalue)
```

Comment:
"So this method ensures that the value is exactly my birth year, right?" - name is quite meaningless